From d45a5293b2f49258224ca6a111ed78b368306138 Mon Sep 17 00:00:00 2001
From: xtex <xtexchooser@duck.com>
Date: Mon, 19 Aug 2024 21:30:30 +0800
Subject: [PATCH] Fix imath target
X-Developer-Signature: v=1; a=openpgp-sha256; l=1163; i=xtexchooser@duck.com;
 h=from:subject; bh=VwXdEFkRxDDQC/Mr75uI+avfq+Rn4i/suq3c7PnlcxM=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGmHPZW3PN68aZGik+DCN1ryXVsfXl1s7TZ3esn94+JZM
 zX3W6g87yhlYRDjYpAVU2QpMmzwZtVJ5xddVi4LM4eVCWQIAxenAEzEezcjw/OQoMLXEWLHpjPr
 /joXYemcpcDaO9dhzRKWz1qnjU/emMrI8D/BSW337WMJ/HkXG8W58+Y4iE7oCXwQZDBv163jTnO
 usQAA
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

Link: AOSC-Dev/aosc-os-abbs runtime-display/imath/autobuild/patches/0002-Export-python3-module.patch
---
 cmake/AlembicPyIlmBase.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/AlembicPyIlmBase.cmake b/cmake/AlembicPyIlmBase.cmake
index 249e084c260f..561a3f2abe19 100644
--- a/cmake/AlembicPyIlmBase.cmake
+++ b/cmake/AlembicPyIlmBase.cmake
@@ -40,7 +40,7 @@ SET(PYTHON_VERSION_MINOR ${Python${PYALEMBIC_PYTHON_MAJOR}_VERSION_MINOR})
 SET(PYTHON_INCLUDE_DIR ${Python${PYTHON_VERSION_MAJOR}_INCLUDE_DIRS})
 
 IF (Imath_FOUND)
-    SET(ALEMBIC_PYILMBASE_PYIMATH_LIB Imath::PyImath_Python${PYTHON_VERSION_MAJOR}_${PYTHON_VERSION_MINOR})
+    SET(ALEMBIC_PYILMBASE_PYIMATH_LIB Imath::imath_python${PYTHON_VERSION_MAJOR})
     MESSAGE(STATUS "Found package Imath using: ${ALEMBIC_PYILMBASE_PYIMATH_LIB}")
 ELSE()
     FIND_PACKAGE(PyIlmBase)
@@ -59,4 +59,4 @@ ELSE()
     ELSE()
         SET(ALEMBIC_PYILMBASE_FOUND 0 CACHE STRING "Set to 1 if PyIlmBase is found, 0 otherwise")
     ENDIF()
-ENDIF()
\ No newline at end of file
+ENDIF()

base-commit: f215bbf47df1035c473464309552a6d1f615f93e
-- 
2.46.0

